components/applications-service/pkg/storage/postgres/services.go

```diff
@@ -167,6 +194,14 @@ func (db *Postgres) GetServices(
 	return services, err
 }
 
+// GetDisconnectedServices returns a list of disconnected services
+func (db *Postgres) GetDisconnectedServices(thresholdMinutes int32) ([]*storage.Service, error) {
+	var services []*storage.Service
+
+	_, err := db.DbMap.Select(&services, fmt.Sprintf(selectDisconnectedServices, thresholdMinutes))
```

Comment:
ℹ️ There's a way to use `ctx` in gorp, too: https://godoc.org/gopkg.in/gorp.v2#DbMap.WithContext

Reply:
I’ll bring this up to the team, you are right and we should use a context.

Reply:
Made a task to fix them all https://github.com/chef/automate/issues/880